### DISCUSSION TITLE
Failed to load latest commit information.

### DISCUSSION THREAD

**Discussion by frogersej123** [community/community#67477] [New to GitHub]
why does my github says " Failed to load latest commit information." from everything i go in to? 
![image](https://github.com/community/community/assets/43281563/7f5d2fe5-b126-46dd-8cc6-037f66886c99)

**Comment by Durgesh4993** [comment]
Forking such a large project might take quite a bit of time.
Interrupting it before completion would mean that HEAD might not have been set.

Try and delete that repo (created in your account), and fork again, this time leaving the fork process running as long as it will need.

Regarding creating a pull request, you do that from a new branch that you have created and pushed to your fork. See 

## couple of tips for PR

Assuming that you have first forked a repo here is what you should do in that fork that you own:

create a branch: isolate your modifications in a branch. Don't create a pull request from master, where you could be tempted to accumulate and mix several modifications at once.
rebase that branch: even if you already did a pull request from that branch, rebasing it on top of origin/master (making sure your patch is still working) will update the pull request automagically (no need to click on anything)
update that branch: if your pull request is rejected, you simply can add new commits, and/or redo your history completely: it will activate your existing pull request again.
"focus" that branch: i.e., make its topic "tight", don't modify thousands of class and the all app, only add or fix a well-defined feature, keeping the changes small.
delete that branch: once accepted, you can safely delete that branch on your fork (and git remote prune origin
. The GitHub GUI will propose for you to delete your branch in your pull-request page.

**Reply by frogersej123** [reply]
its not mine repo its somelse's

**Comment by Durgesh4993** [comment]
The error message "Failed to load latest commit information" typically occurs when there is an issue with the repository or the connection to the repository. Here are a few steps you can take to troubleshoot and resolve the error:

Check the repository status: Ensure that the repository hosting service (e.g., GitHub, GitLab, Bitbucket) is functioning properly and accessible. Visit the repository's web interface and verify if you can view the commits, branches, and other information. If the repository is experiencing issues, you may need to wait until the service is restored.

Verify your network connection: Confirm that you have a stable internet connection. The error may occur if your network connection is intermittent or experiencing disruptions. Try accessing other websites or repositories to ensure your internet connection is working correctly.

Clear browser cache: The error might be related to a caching issue in your browser. Clear the cache for the specific repository's web page or try opening the repository in a different web browser. Alternatively, you can also try using incognito/private browsing mode to rule out any caching problems.

Check repository permissions: Ensure that you have the necessary permissions to access the repository and view its commit information. If you are accessing a private repository, make sure you are logged in with the correct account and have the appropriate access rights.

Temporary server issues: Sometimes, repository hosting services may experience temporary issues or maintenance periods that can affect the availability of commit information. Check the status page or official support channels for the repository hosting service to see if there are any reported issues. If there are, you may need to wait until the service is fully restored.

Contact repository support: If the issue persists or if you suspect there might be an issue specific to the repository, reach out to the support team or community forums of the repository hosting service. They can provide further assistance and investigate any potential problems with the repository.

By following these steps, you should be able to troubleshoot the "Failed to load latest commit information" error and regain access to the commit information in your repository.

**Reply by Durgesh4993** [reply]
@frogersej123 you can try this

**Reply by frogersej123** [reply]
does not help, i says the same error in every repo i go into

**Comment by Durgesh4993** [comment]
@frogersej123 first check your interner, and log out with your id and then login

**Reply by frogersej123** [reply]
my internet is fine

**Reply by Durgesh4993** [reply]
log out and then log in

**Reply by Durgesh4993** [reply]
and make difference betweek 5 min to login

**Comment by Durgesh4993** [comment]
Hope this work

**Reply by frogersej123** [reply]
dont work

**Reply by CarolineLFye** [reply]
Hey I am getting this same issue today 09/18/2023. Could this be a git glitch? did you figure out anything?

**Reply by frogersej123** [reply]
Nope mine does still not work

**Comment by flp511** [comment]
you needs to deal with 'Pull requests'

**Comment by bHabawel** [comment]
Hello, I guess it has to do with the pull request.
